test/unit/lib/actions/entry-transform.spec.ts

```diff
@@ -99,7 +99,7 @@ describe('Entry Action', function () {
         'hawaii': 'haukea'
       }
     })
-    expect(batches[0].requests[2].data.fields).to.eql({
+    expect(batches[0].requests[1].data.fields).to.eql({
```

Comment:
In this test we were updating two entries, since we were publishing by default, there were 4 requests[ (1 update + 1 Publish) * 2], now we just have two update requests

that's why I updated the assertion 👍